src/commands/coverage_publish.yml

```diff
@@ -12,64 +12,74 @@ parameters:
     description: The name of the tag to use for the coverage report.
     type: string
     default: ""
-  report_format:
```

Comment:
We had `report_format` previously but we don't seem to be doing anything with it, so just removed it.

Reply:
The goal with format is that the CLI should accept `--format` and the GH Action and the Orb accept `format` as input